Eevee.Sleep.Bot/Workers/Crawlers/OfficialsiteAnnouncementCrawler.cs

```diff
@@ -37,20 +37,20 @@ ILogger<InGameAnnouncementCrawler> logger
         return await Task.WhenAll(detailTasks);
     }
 
-    private static async Task SaveDetailsAndHistories(IEnumerable<InGameAnnouncementDetailModel> details) {
-        var existedDetails = InGameAnnouncementDetailController.FindAllByIds(details.Select(x => x.AnnouncementId));
+    private static async Task SaveDetailsAndHistories(IEnumerable<OfficialSiteAnnouncementDetailModel> details) {
```

Comment:
Can't be `IEnumerable<OfficialSiteAnnouncementDetailModel>` because it's enumerated multiple times in this method.